BuildingSync.xsd

```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="DemisingAboveGradeWallArea" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Above grade demising wall area. (ft2)</xs:documentation>
```

Comment:
We might want to reverence the ASHRAE standard here. If you search in the schema there are a few places where we do this

Reply:
> We might want to reverence the ASHRAE standard here. If you search in the schema there are a few places where we do this

Could you point me to those places?

Reply:
Here's an example: https://github.com/BuildingSync/schema/blob/458dcc218d771d2520cd27aa7a87b63fac560e14/BuildingSync.xsd#L4106